scripts/events_module/short/condition_events.py

```diff
@@ -942,32 +942,32 @@ def determine_retirement(cat, triggered):
                     continue
 
                 if cat.permanent_condition[condition]["severity"] == "severe":
-                    # Higher changes for "severe". These are meant to be nearly 100% without
+                    # Higher chances for "severe". These are meant to be nearly 100% without
                     # being 100%
                     retire_chances = {
-                        "newborn": 0,
-                        "kitten": 0,
-                        "adolescent": 50,  # This is high so instances where an cat retires the same moon they become an apprentice is rare
-                        "young adult": 10,
-                        "adult": 5,
-                        "senior adult": 5,
-                        "senior": 5,
+                        CatAgeEnum.NEWBORN.value: 0,
```

Comment:
Does this need to have `value` explicitly identified if we're only ever comparing it to other StrEnums?